test/library/standard/GMP/elliot/check-gmp-none-compiler-error.good

```diff
@@ -0,0 +1 @@
+$CHPL_HOME/modules/standard/GMP.chpl:1226: error: GMP support not compiled in
```

Comment:
The bad/wrong part of this PR is that we're reporting the internal GMP line number, not the user line number. I think this is because the compilerError is just top-level in the module, and not inside a user-callable function. I'm not sure I understand the compilers handling of error reporting to solve this myself though.

Reply:
That would probably require recognizing `use`s as contributing to the module init call chain, which is something our compiler doesn't do strongly right now.  If there's a way to remove the line number completely from the output, I would consider that a suitable compromise (though I'm not sure that's possible)

Reply:
Does the Regexp module have the same issue?  compilerError() takes an errorDepth argument which should permit you to pop up a few levels of callchain potentially, but I can never quite remember how it works nor how well it works.  See test/param/diten/errorDepth.chpl, e.g.

Reply:
Regexp has the compilerError inside the `compile()` proc, since you have to compile a regex in order to use it, so it correctly reports the user error line.  For gmp we _could_ make wrappers for all the gmp init routines and put the compilerError in there, but there's a lot of init routines (and currently they're all just naked externs)

I tried an errorDepth of 0, but that didn't change anything since I think that's the default

Reply:
I'm OK with putting this in as-is and striving to improve it in the future.